app/controllers/application_controller.rb

```diff
@@ -48,7 +48,15 @@ def peek_enabled?
   private
 
   def current_user
-    @current_user ||= User.find_or_create_by(email_address: session[:user]['email']) if session[:user] && session[:user]['email']
+    @current_user ||= find_current_user
+  end
+
+  def find_current_user
+    if ENV['FORCE_USER_ID']
```

Comment:
This is a development hack.

In order simulate what a user sees, w/o being able to authenticate as that user, you can set the `FORCE_USER_ID` environment variable to a User id value when starting the rails server. E.g.:
```
FORCE_USER_ID=1234 rails server
```

This is most useful in dev when using a db snapshot from production, when trying to re-create what a user is seeing. Since we can't authenticate as that user, we can mimic the login by forcing the server to treat each request as if the current_user is someone other than yourself.